CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Removed
 
 ### Fixed
+- CSS rules with property values containing newlines are dropped.
```

Comment:
Could you also please link the PR (as in the other changelog entries)? Then we don't need to edit the changelog to merge this PR.

Reply:
Could you please reword this so it's about what's fixed now, for example like this (also in the commit message):

Allow property values with newlines

Reply:
Done & done.